### DISCUSSION TITLE
Generate and import development data for user

### DISCUSSION THREAD

**Discussion by chamalis** [rotki/rotki#7883] [Q&A]
Hello all

Is there a way to populate `develop_data/users/<user>/rotkehlchen.db` with development data for user information ?
That way someone without assets or/and exchanges API keys could work on with the app.
If not, any alternative ideas?

Thanks

**Comment by yabirgb** [comment]
Hello @chamalis ! If you mean that if we have prepared data to populate the database with that on demand we don't have such functionality avaliable. What we genearly do is to copy our production information (the db from `data/users/<user>/rotkehlchen.db`) and work with that. If you need assistance developing something you can jump to our discord https://discord.rotki.com/ where we can help you better

**Reply by chamalis** [reply]
Hey @yabirgb, thank you for your prompt reply.
Unfortunately, I don't have and can't make a discord account right now. 
I will try to generate some random data (because I don't have like real assets), in order to work on some issues that need accounting data, like #1215 or #7675.
